### ISSUE TITLE
Replicate fine-tuning results

### ISSUE THREAD

**Issue by Yuliang-Zou** [TengdaHan/MemDPC#7]
Hi, 

I wonder how I should set the training config so that I can reproduce the full network fine-tuning results on HMDB51 in Table1 (visual stream only, 41.2). Also, I wonder how I should adjust the training schedule to get the results in Figure 5 (visual stream only)?

Thank you and look forward to your response!

**Comment by TengdaHan** [comment]
Hi! 

* reproduce HMDB51 results -- Actually I left the scheduler [here](https://github.com/TengdaHan/MemDPC/blob/master/eval/test.py#L121). I trained for 300 epochs in total with 224 resolution. For batch size, I use 8 samples per GPU and trained on 2 GPUs -- but batch size (varies between 8-16) will not affect much from my experience. 

* Figure 5 - This is the data efficiency experiments. First I randomly sampled 10%, 20% and 50% subset of UCF101. Then I use the same setting as Table 1 (use 128 resolution), except I trained for 350 epochs to save time, and reduce lr once on 300 epochs. 

Let me know if you have other questions.
